examples/zstd/fileio.c

```diff
@@ -0,0 +1,2791 @@
+/*
```

Comment:
We don't need this complicated of an example program. Just something that includes the header and uses a library function would be enough.